src/core/hooks.js

```diff
@@ -48,11 +48,11 @@ export function buildHook (moduleContainer, options) {
   // Add vue-i18n-loader if applicable
   if (options.vueI18nLoader) {
     moduleContainer.extendBuild(config => {
-      const loaders = config.module.rules.find(el => el.loader === 'vue-loader').options.loaders
-      if (loaders) {
+      const vueLoader = config.module.rules.find(el => el.loader.includes('vue-loader'))
+      if (vueLoader && vueLoader.options && vueLoader.options.loaders) {
         // vue-loader under 15.0.0
         /* istanbul ignore next */
-        loaders.i18n = '@intlify/vue-i18n-loader'
+        vueLoader.options.loaders.i18n = '@intlify/vue-i18n-loader'
```

Comment:
It would be nice if you use `require.resolve` here to resovle `@intlify/vue-i18n-loader` on behalf of module (otherwise we depend on implicit `node_modules` structure)

Reply:
Done.